File: 2715-KItemsWithTheMaximumSum/2715-KItemsWithTheMaximumSum.py
# Last updated: 7/31/2025, 5:28:04 PM
class Solution:
    def kItemsWithMaximumSum(self, numOnes: int, numZeros: int, numNegOnes: int, k: int) -> int:
        s=numOnes
        b=numZeros
        if(k<=s):
            return k
        if(k>s and k<=s+b):
            return s
        if(k>s+b):
            a=k-(s+b)
            return s-a
        

File: 2715-KItemsWithTheMaximumSum/2715-KItemsWithTheMaximumSum.java
// Last updated: 7/31/2025, 5:28:08 PM
class Solution {
    public int kItemsWithMaximumSum(int numOnes, int numZeros, int numNegOnes, int k) {
        int s=numOnes;
        int b=numZeros;
        int y=0;
        if(k<=s){
            y=k;
        }
        if(k>s && k<=s+b){
            y=s;
        }
        if(k>s+b){
            int a=k-(s+b);
            y= s-a;
        }
        return y;
    }
}